logging/telemetry.go

```diff
@@ -109,14 +109,14 @@ func ReadTelemetryConfigOrDefault(dataDir string, genesisID string) (cfg Telemet
 
 		configPath, err = config.GetConfigFilePath(TelemetryConfigFilename)
 		if err != nil {
-			// In this case we don't know what to do since we couldn't
-			// create the directory.  Just create an ephemeral config.
+			// In this case we don't know what to do since we couldn't read
+			// from the global directory. Just create an ephemeral config, if
+			// the data directory was provided it can be persisted there later.
 			cfg = createTelemetryConfig()
```

Comment:
this winds up being the same assignment statement as the next `err != nil` statement